css/main.css

```diff
@@ -159,6 +159,919 @@ nav {
     margin: 0 0px -100px -100px;
 }
 
+/* OUR SERVICES */
+
+.services {
+    margin: 127px 60px 0 60px;
+    background-color: var(--GREY) ;
+}
+
+.services_head {
+    padding: 100px 90px 0 90px;
+    color: var(--GREEN);
+}
+
+.services h2 {
+    width: 675px;
+    margin: 0 90px 0 90px;
+}
+
+.services_sub {
+    width: 595px;
+    margin: 0 90px 0 90px;
+}
+
+.services_grid {
+    display: flex;
+    padding: 60px 90px 290px 90px;
+}
+
+.box {
+    width: 260px;
+    margin-right: 33px;
+    background-color: var(--WHITE);
+    padding: 38px 0 0 20px;
+}
+
+.services_bold {
+    font-weight: bold;
+    font-weight: 1.125rem;
+    margin: 16px 0 0 0;
+}
+
+.box p {
+    width: 213px;
```

Comment:
try to avoid giving things explicit widths

Reply:
so i should use "min" instead?